client/jetpack-connect/auth-logged-out-form.jsx

```diff
@@ -108,7 +112,7 @@ class LoggedOutForm extends Component {
 				{ this.renderLocaleSuggestions() }
 				<AuthFormHeader authQuery={ this.props.authQuery } />
 				<SignupForm
-					redirectToAfterLoginUrl={ this.getRedirectAfterLoginUrl() }
+					redirectToAfterLoginUrl="!! THIS PROP IS REQUIRED BUT UNUSED !!"
```

Comment:
Yeah… worth having a look at what's going on here.

See p1513354730000161-slack-jetpack-conn-services

Reply:
Good idea 👍 

I've been looking into how other parts of Calypso do this but it's not clear to me yet. Hopefully we can make this a bit less confusing.

Reply:
Added a component-level docblock in 106537b, let me know if you think it needs work.